src/MessengerPlugin.jsx

```diff
@@ -31,14 +34,25 @@ export default class MessengerPlugin extends Component {
     }
 
     initFacebookSDK() {
-        const {FB, appId} = this.props;
+        const {FB, appId, onRender, onClick, onNotYou} = this.props;
 
         if (FB) {
             FB.init({
                 appId,
                 xfbml: true,
                 version: 'v2.6'
             });
+
+            FB.Event.subscribe('send_to_messenger', function(e) {
```

Comment:
The lib should unsubscribe from the event when it unmounts too to avoid ghost listeners.